README.md

```diff
@@ -21,15 +18,38 @@ conceptual documentation on your Drupal site.
  * To submit bug reports and feature suggestions, or to track changes:
    https://www.drupal.org/project/issues/devportal
 
+ * You can add the 'FAQ' submodule to your Devportal. Questions can be grouped by topics and these topics can be used as filters on the FAQ listing page.
+
+ * We recommend you to enable the 'Guides' module, which gives the ability for writers to upload docs in Markdown. It is a useful tool for content editors. Read more here:
+https://github.com/Pronovix/devportal-drupal-module/blob/8.x-2.x/modules/guides/README.md
```

Comment:
You can use `here` as link title. Also, can you try adding this URL as a relative one? I'm not sure that it's even possible in GitHub, but it would be great to leave the branch name out from the URL.